ai/yolo/auto_labeling_script/main.py

```
@@ -0,0 +1,206 @@
+        print(f"Uwaga: crop ma rozmiar {cropped_img.shape[:2]}, a nie (640, 640)")
+
+    best = find_best_square_roi(
+        full_img,
+        cropped_img,
+        min_size=min_size,
+        max_size=max_size,
+        step=step
+    )
+
+    if best is None:
+        raise RuntimeError("Nie znaleziono dopasowania.")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)